presentation/components/src/test/index.ts

```diff
@@ -0,0 +1,8 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+export * from "./_helpers/Properties";
+export * from "./_helpers/UiComponents";
+import { PRESENTATION_TREE_NODE_KEY } from "../presentation-components/tree/Utils";
```

Comment:
Why is this necessary? Consumers should be able to import from `presentation-components` barrel.

Reply:
My guess is so that test exports can only be imported through `@itwin/presentation-components/test`, and I like it.

Reply:
I agree regarding the tests' barrel file. My question (which I agree is definitely not clear) is specifically about the `PRESENTATION_TREE_NODE_KEY` - it's part of the library and is exported through the package's barrel. We shouldn't need to re-export it from here.

Reply:
This actually wasn't being exported in the pkg's main barrel, i assume because it was marked as internal.
I can export it in the main barrel if you'd like.

![image](https://user-images.githubusercontent.com/11051042/135628621-53ba1796-a14c-4bca-aad6-cd5609ff5a1d.png)
The const was defined in `presentation/components/src/presentation-components/tree/Utils.ts`

Reply:
Yes, please add it to the main barrel

Reply:
done